### DISCUSSION TITLE
Testimonials section- moivng through the testimonials

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#145] [Feedback]
There is no intuitive control for scrolling through the testimonials. 
Now it is set to automatically scroll through the testimonials
If someone wants to read all the testimonials or they were reading while it scrolled to the next one then they do not have any intuitive or obvious mechanism to go back to that testimonial.

On desktop it is working with mouse scroll button or clicking on the small dots below the testimonial on mobile i could not figure out a way except for being patient and waiting.

**Comment by an-nadaa** [comment]
The bug we see with mobiles is probably due to the limitations of the library we use for implementing the carousel functionality. it probably doesn't support touch screen feature.

With that said. i am planing to revamp the gallery and testimonial section. will work on two new sections soon.
